_sass/_page.scss

```diff
@@ -61,6 +61,12 @@
     font-size: 0.8rem;
   }
 
+  blockquote {
+    p {
+      margin: 1em 1em 1em;
```

Comment:
v minor nit: shorthand `margin: 1em;` should have the same effect here.

Reply:
Isn't that the same as `margin: 1em 1em 1em 1em`? I don't want the left margin to be `1em`

Reply:
`margin: 1em 1em 1em;` is also the same as `margin: 1em 1em 1em 1em;`. when you drop the left margin value it takes the value from the right margin unless you override it elsewhere.

Reply:
woah